ai2_internal/timo/vila/integration_test.py

```diff
@@ -0,0 +1,94 @@
+"""
+Write integration tests for your model interface code here.
+
+The TestCase class below is supplied a `container`
```

Comment:
can u include instructions somewhere for how to run these tests locally, including obtaining the container that this should point to?

Reply:
Sure. This is documented in the `tt` user guide over here: https://github.com/allenai/timo/blob/main/docs/timo-tools/userguide.md#step-5-add-some-integration-tests

But I can make a reference to those docs in this docstring.